docs/DEPLOYMENT.md

```diff
@@ -294,18 +294,54 @@ Update your GitHub OAuth created with the UI route in place of `Homepage URL` an
 
 ## Add resources in DB
```

Comment:
this is simplified now

- user has already added themselves in hub config
- login into hub UI
- take token from UI
- call the refresh api